### PR TITLE
Project news site v 2

### PR DESCRIPTION
**Pull request by Ruterryter**
Hi, 
Here comes my try at this project. It's been a struggle...

Link: 
https://thirsty-curie-69d291.netlify.com/

Best
Hanna

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,166 @@
...
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins|Roboto&display=swap" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://kit.fontawesome.com/41a2006263.js" crossorigin="anonymous"></script>
+
+  <!-- dont forget the meta tag for responsiveness -->
```
Comment (ericaglimsholt) [inline code/index.html]:
```suggestion

```

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,166 @@
...
+      <a href="#footer">Contact</a>
+    </nav>
+
+    <div>
+      <img class="logosmall" src="wine.svg" alt="logo">
+    </div>
+
+    <div class="name">
+      <p class="name">TableTop</p>
+    </div>
```
Comment (ericaglimsholt) [inline code/index.html]: How come this two elements has the same class name?

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,166 @@
...
+  </header>
+
+
+  <!-- The big news content -->
+
+  <section class="big-news-container">
+    <div class="big-news">
+      <img src="images/insta_citron_skumpa_mork_haller.jpg" class="big-news-picture" alt="Champagne image">
```
Comment (ericaglimsholt) [inline code/index.html]: A screen reader for example will read "image champagne" so there's no need to add "image" to an alt text
```suggestion
      <img src="images/insta_citron_skumpa_mork_haller.jpg" class="big-news-picture" alt="Champagne">
```

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,166 @@
...
+
+  </main>
+
+  <footer>
+    <div id="footer">
+      <div class="contact">
+        <a href="mailto:tabletop@tabletop.se">Send us an email</a>
+      </div>
+
+      <p class="p-footer">Design by Hanna Rüter</p>
```
Comment (ericaglimsholt) [inline code/index.html]: Do you need this class? Here you could get this element by `footer p {}` in the css

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,320 @@
...
+}
+
+a:link {
   color: #fff;
+  text-decoration: underline;
+}
+
+a:hover {
+  color:#ff66a9;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,320 @@
...
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background-color: white;
+}
+
+.name {
+  font-size: 24px;
+  margin-right: 20px;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

**Comment by Ruterryter** [comment]
Thanks for the input! I have now done the suggested changes. 
Best Hanna

**Pull request closed without merging**